go/contacts/cache.go

```diff
@@ -157,33 +153,42 @@ func (c *CachedContactsProvider) LookupAll(mctx libkb.MetaContext, emails []keyb
 		conCache = newLookupResultCache()
 	} else {
 		mctx.Debug("Fetched cache, current cache size: %d", len(conCache.Lookups))
+		conCache.Version.Major = cacheCurrentMajorVersion
+		conCache.Version.Minor = cacheCurrentMinorVersion
 	}
 
 	var remainingEmails []keybase1.EmailAddress
 	var remainingNumbers []keybase1.RawPhoneNumber
 
 	mctx.Debug("Populating results from cache")
 
+	// Map of keys of new unresolved cache entries, to set ExpireAt value after
```

Comment:
```suggestion
	// List of keys of new unresolved cache entries, to set ExpireAt value after
```

Reply:
Also it's not just unresolved entries, stale resolved entries are looked up too right?

Reply:
yes, thanks